draft-ietf-rats-eat.md

```diff
@@ -1171,11 +1171,12 @@ See {{common-types}} for OID and URI encoding.
 
 ### intuse (Intended Use) Claim
 
-EAT's may be used in the context of several different applications.  The "intuse"
+EATs may be used in the context of several different applications.  The "intuse"
 claim provides an indication to an EAT consumer about  the intended usage
 of the token. This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT.
 5 possible values for "intuse" are currently defined indicated 5 different intended use indications, but additional values above 5 can
```

Comment:
"additional values above 5" reads slightly awkwardly to me. Maybe just "additional values"?

Reply:
@quic-jodonogh change made.